governance.md

```diff
@@ -84,7 +84,16 @@ The initial composition of the SC is self-defined and is made of 10 members:
 
 #### Selection
 
-The initial SC members will be responsible for defining how and when the SC membership can be expanded or renewed.
+SC membership can be expanded by a unanimous vote of the commitee.
+Individuals wishing to join the SC should make a PR to this file updating the above list.
+The PR message should include a brief background of the proposed member.
+There are no eligibility requirements for membership.
+The SC committee will mainly consider community contributions and influence when voting on new members.
+This could include code contributions to the project, maintainership of project components, and participation in community meetings among other things.
+
+An existing SC member can remove themselves from the committee by making a PR to this file.
+
+The SC will develop a protocol for renewal of committee membership.
```

Comment:
Chatting on with @larrydewey and @sameo , An idea to gently extend this is perhaps have a date at which the SC dissolve and reforms. We don't necessarily need to have elections at present as I don;t feel we have any issues regarding size of SC yet?
Perhaps our general principles about ensuring diverse representation from companies/views/contribution areas?
And our general principle to push contentious discussions into the weekly well attended meetings?

Reply:
But I am happy to approve what is here and we treat as a living document think about a further PR to add more.

Reply:
An opt in process when SC reforms to start with for both existing SC and wider community members, but SC is more admin/process longer term thoughts. But discussions decisions are driven by the weekly community calls.